src/status_im/contexts/shell/activity_center/notification/contact_verification/view.cljs

```diff
@@ -105,7 +105,7 @@
   (rf/dispatch [:activity-center.notifications/mark-as-read id]))
 
 (defn- swipeable
-  [{:keys [active-swipeable extra-fn notification replying?] :as props} child]
```

Comment:
You can safely ignore this file because contact verification notifications are not being actively tested for more than a year. But I still updated the code. We may end up feature flagging this type of notification for 2.30.